carrierwave-postgresql.gemspec

```diff
@@ -16,7 +16,11 @@ Gem::Specification.new do |s|
   s.add_dependency "carrierwave"
 
   s.add_development_dependency "activerecord", "~> 4.0.1"
-  s.add_development_dependency "pg"
+  if defined?(JRUBY_VERSION)
```

Comment:
I'm not sure if this is the proper way to handle platform dependent dependencies.
It seems that this was evaluated once you ran the bundle install.
To select dependencies at runtime you need to use bundler, it seems that the information in [this post](http://blog.mattwynne.net/2011/04/26/targeting-multiple-platforms-jruby-etc-with-a-rubygems-gemspec/) is still accurate